DEVELOPMENT.md

```diff
@@ -55,6 +56,50 @@ Nonetheless, input needs to be validated to catch mistakes:
   refuses to send messages that are larger
   (https://godoc.org/google.golang.org/grpc#MaxSendMsgSize)
 
+Branching strategy
+==================
+
+The `master` branch is the main branch. It is guaranteed to have
+passed full CI testing. However, it always uses the latest Clear Linux
+for building container images, so changes in Clear Linux can break the
+building of older revisions.
```

Comment:
Another approach would be to lock Clear Linux also on the `master` branch. But that means that merging new changes from `devel` becomes more complicated:
- run `hack/update-clear-linux-base.sh` on the `devel` branch
- commit locally
- run the tests
- merge master with that local commit

It actually should be fairly doable, I just don't know whether it is worth it. In practice, Clear Linux updates haven't been problematic for building PMEM-CSI (although Go 2.0 entering Clear could change that). It's Kubernetes on Clear which is problematic, and that currently doesn't get addressed by `hack/update-clear-linux-base.sh` but rather by the `TEST_CLEAR_LINUX_VERSION` values in our Jenkinsfile.